### DISCUSSION TITLE
v1.98.4

### DISCUSSION THREAD

**Discussion by cyring** [cyring/CoreFreq#509] [Show and tell]
# [Build]
* Prevent a collision with macro `WRMSRNS` within kernel `6.11`
# [x86_64]
* [Virtualization] Switch to the HCF or VP_RUNTIME counters depending on the detected hypervisor

<hr /><em>This discussion was created from the release <a href='https://github.com/cyring/CoreFreq/releases/tag/1.98.4'>v1.98.4</a>.</em>

**Comment by cyring** [comment]
@bubbleguuum @inglor @ich777

Hello,

For your information, this _CoreFreq_ revision is necessary to build against latest RC of Kernel `6.11`

Regards

**Comment by bubbleguuum** [comment]
Thanks for the info. Submitted to openSUSE TW which is timely since it will soon get kernel 6.11.
